docs/source/terms_and_model.rst

```diff
@@ -1411,37 +1286,84 @@ Absolute copy number counts may not be smaller than zero.
      - Type
      - Limits
      - Description
+   * - value
+     - number
+     - 1..1
+     - A simple number value
    * - type
      - string
      - 1..1
-     - MUST be "AbsoluteCopyCount"
+     - MUST be "Number"
+
+.. _DefiniteRange:
+
+DefiniteRange
+#############
+
+**Computational Definition**
+A bounded, inclusive range of numbers.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
    * - min
-     - integer
+     - number
      - 1..1
-     - minimum value; inclusive
+     - A simple number value representing the range minimum
    * - max
-     - integer
+     - number
      - 1..1
-     - maximum value; inclusive
-
-**Implementation Guidance**
+     - A simple number value representing the range maximum
+   * - type
+     - string
+     - 1..1
+     - MUST be "DefiniteRange"
 
-* If both ``min`` and ``max`` MUST satisfy ``min
-  <= max``.
-* If ``min == max``, then the range specifies a single numeric amount.
-* Both ``min`` and ``max`` MUST be non-negative
+.. _IndefiniteRange:
 
+IndefiniteRange
+###############
 
-**Examples**
+**Computational Definition**
+An indefinite range represented as a number and associated comparator.
+The bound operator is interpreted as follows: '>=' are all values
+greater than and including the value, '<=' are all numbers less than and
+including the value.
 
-.. parsed-literal::
+**Information Model**
 
-   {
-     "max": 4,
-     "min": 0,
-     "type": "AbsoluteCopyCount",
-   }
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
 
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - value
+     - number
+     - 1..1
+     - A simple number value representing a range boundary
+   * - comparator
+     - number
+     - 1..1
+     - MUST be one of "<=" or ">=", indicates in which direction
```

Comment:
I think its OK to leave out > and < since the user can simply increase or decrease the value number to make it work. If that's our rationale for leaving out the > and < options then we should probably clarify that in the Impl Guidance section 
```
**Implementation Guidance**
* `>` and `<` options were intentionally omitted. Increasing or decreasing the numeric value to always fulfill the `>=` and `<=` comparators will benefit consistent use and improve normalization.
* The `=` comparator was intentionally omitted. Users should use the `Number` or `DefiniteRange` types if exact values are desired.

Reply:
Right. The reason for not having both the inclusive and exclusive forms is that it creates ambiguity. "copy > 2" probably means the same thing as "copy >= 3".